guidelines/monai-application-package.md

```diff
@@ -1,498 +1,650 @@
+| `models`                                                | No       | N/A                     | array       | array of objects          | Array of objects which describe models in the package.                                                        |
+| `models[*].name`                                        | **Yes**  | N/A                     | string      | string                    | Name of the model.                                                                                            |
+| `models[*].path`                                        | No       | N/A                     | string      | Relative file-system path | File-system path to the folder which contains the model that is relative to the value defined in `modelRoot`. |
+| `resources`                                             | No       | N/A                     | object      | object                    | Object describing resource requirements for the Application.                                                  |
+| `resources.cpu`                                         | No       | `1`                     | decimal (2) | number                    | Number of CPU cores required by the Application or the Fragment.                                              |
+| `resources.cpuLimit`                                    | No       | N/A                     | decimal (2) | number                    | The CPU core limit for the Application or the Fragment. (1)                                                   |
+| `resources.gpu`                                         | No       | `0`                     | decimal (2) | number                    | Number of GPU devices required by the Application or the Fragment.                                            |
+| `resources.gpuLimit`                                    | No       | N/A                     | decimal (2) | number                    | The GPU device limit for the Application or the Fragment. (1)                                                 |
+| `resources.memory`                                      | No       | `1Gi`                   | string      | memory size               | The memory required by the Application or the Fragment.                                                       |
+| `resources.memoryLimit`                                 | No       | N/A                     | string      | memory size               | The memory limit for the Application or the Fragment. (1)                                                     |
+| `resources.gpuMemory`                                   | No       | `1Gi`                   | string      | memory size               | The GPU memory required by the Application or the Fragment.                                                   |
```

Comment:
Is `1Gi` a correct default for `gpuMemory` if the default value for `resources.gpu` is `0`?

Reply:
Updating to `N/A`